.circleci/config.yml

```diff
@@ -0,0 +1,135 @@
+#
+# Copyright 2016-2018 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/openjdk:8-jdk
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: maven-dependencies-{{ checksum "pom.xml" }}
+
+      - run:
+          name: Download dependencies
+          command: |
+            sudo apt-get install xsltproc
+            ./build-support/go-offline.sh
+      - save_cache:
+          key: maven-dependencies-{{ checksum "pom.xml" }}
+          paths:
+            - ~/.m2
+
+      - run:
+          name: Tests
+          command: ./mvnw clean verify
+
+      - run:
+          name: Collect test reports
+          command: |
+            mkdir -p /tmp/test-reports/unit-tests/
+            find . -type f -regex ".*/target/surefire-reports/.*xml" -exec cp {} /tmp/test-reports/junit/ \;
```

Comment:
`/tmp/test-reports/junit/` should be `/tmp/test-reports/unit-tests/`

Reply:
Ah. The original was using `junit`, and I managed to mangle things here. Do we keep the naming consistent between `zipkin-gcp` CircleCI 1.0 and CircleCI 2.0, or do we keep the naming consistent between different OpenZipkin projects?

Reply:
I would personally prefer consistency across projects, but I can understand the argument either way. Particularly in cases where we only have one type of tests, it is less important. I just thought it was nice to distinguish between unit and integration test results, and not focus on the underlying technology, which is the same in the case of JUnit.